go.mod

```diff
@@ -186,3 +187,5 @@ require (
 	golang.org/x/exp v0.0.0-20230425010034-47ecfdc1ba53
 	gopkg.in/yaml.v3 v3.0.1
 )
+
+replace github.com/openvex/go-vex => github.com/openvex/go-vex v0.2.1-0.20230519031452-e31eaf94ec95
```

Comment:
hmm, could we just upgrade to `go-vex` or is there a breaking diamond dependency ?

Reply:
We need a v0.2.1 release of `go-vex` for this to work properly. I'll see if we can push for that to happen soon.

Reply:
We can create an issue to track this and update once it's released. We can resolve this conversation to merge the PR.

Reply:
Tracked in https://github.com/guacsec/guac/issues/1017